Sources/BeeTracking/OISTBeeVideo+Batches.swift

```diff
@@ -62,6 +62,84 @@ extension OISTBeeVideo {
     return statistics.normalized(stacked)
   }
 
+  /// Returns a batch of locations of background
```

Comment:
I thought we already had this type of functionality. Where is the old code, and how is it different? No copy/pasta please!